src/views/LoginView.vue

```diff
@@ -4,7 +4,7 @@
       <img id="login-image" src="../assets/img/bier.png" alt="logo" />
       <h1>{{ $t('login.SudoSOS Login') }}</h1>
       <Button id="login-gewis-button" @click="loginViaGEWIS" severity="success"
-        ><img id="gewis-branding" src="../assets/img/gewis-branding.svg" alt="GEWIS" />{{
+      ><img id="gewis-branding" src="../assets/img/gewis-branding.svg" alt="GEWIS" />{{
```

Comment:
moet je niet @ gebruiken? is dit hoe je assets er uit haalt? lijkt me niet toch.

Reply:
Wou ik nog een keertje met je naar kijken, want WebStorm snapt geen klote van @ syntax.